cli/lib/nf3.mjs

```diff
@@ -1066,6 +1066,7 @@ class Nf3 {
       fee,
     });
     logger.debug(`Proposer /update response ${res}`);
+    return res?.data;
```

Comment:
:memo:

Have updated tests with call to `/register`, so that these "await" for the register tx to be mined (event `NewCurrentProposer` can only be guaranteed in an env with 1 proposer).

Does `nf3` need to be updated? Tests using `nf3.registerProposer` seem to be passing anyway.
Keep an eye.